// ERR

fun test(String s) {
    print(s);
}

func main() {
    test()
}